halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/DeploymentStrategy.java

```diff
@@ -0,0 +1,16 @@
+package com.netflix.spinnaker.halyard.deploy.spinnaker.v1.service;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Data;
+
+@Data
```

Comment:
I'd suggest either using `@Data` to generate accessors or leaving the fields public, but not both.

Reply:
Showing my lack of experience with Java/lombok here :) I'll get this fixed. Thanks!

Reply:
Fixed.